Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -17432,7 +17431,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "echo \"Preprocessing sandbox\"\nScripts/generate-derived-sources.sh sandbox-profiles-ios\nmkdir -p ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.adattributiond.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.webpushd.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.GPU.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.Networking.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebAuthn.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebContent.sb ${DSTROOT}/${INSTALL_PATH}\n";
+			shellScript = "echo \"Preprocessing sandbox\"\n\nset -e\n\nScripts/generate-derived-sources.sh sandbox-profiles-ios\nmkdir -p ${DSTROOT}/${INSTALL_PATH}\n\nSANDBOX_GPU=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.GPU.sb\nSANDBOX_NETWORKING=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.Networking.sb\nSANDBOX_WEBCONTENT=${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebContent.sb\n\nsbutil compile ${SANDBOX_GPU} > /dev/null\nsbutil compile ${SANDBOX_NETWORKING} > /dev/null\nsbutil compile ${SANDBOX_WEBCONTENT} > /dev/null\n\ncp ${SANDBOX_GPU} ${DSTROOT}/${INSTALL_PATH}\ncp ${SANDBOX_NETWORKING} ${DSTROOT}/${INSTALL_PATH}\ncp ${SANDBOX_WEBCONTENT} ${DSTROOT}/${INSTALL_PATH}\n\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.adattributiond.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.webpushd.sb ${DSTROOT}/${INSTALL_PATH}\ncp ${BUILT_PRODUCTS_DIR}/DerivedSources/WebKit/com.apple.WebKit.WebAuthn.sb ${DSTROOT}/${INSTALL_PATH}\n";
```

Comment:
Do you need to tell the build script to include the `/usr/local/share/sandbox/profiles/embedded/imports/` path anywhere?

When I do the equivalent thing from the command line I have to have something like this in my shell environment:

```
IOS_SANDBOX_IMPORT_PATH="$DEV_ENVIRONMENT_PATH/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS$CURRENT_IOS_DEV_TARGET.Internal.sdk/usr/local/share/sandbox/profiles/embedded/imports/
```